SPECS/cassandra-cpp-driver/cassandra-cpp-driver.spec

```diff
@@ -0,0 +1,61 @@
+Summary:        DataStax C/C++ Driver for Apache Cassandra and DataStax Products
+Name:           cassandra-cpp-driver
+Version:        2.16.0
+Release:        1%{?dist}
+Epoch:          1
+License:        Apache 2.0
```

Comment:
Nit: Can we change this to `ASL 2.0`? We generally try to match the "Short Name" from [Fedora's license list](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Software_License_List)